scissors/scissors_game/main.c

```diff
@@ -1,8 +1,19 @@
 // SPDX-License-Identifier: GPL-2.0
 #include <stdio.h>
+#include <string.h>
+#include "scissors.h"
 
 int main(void)
 {
-	printf("Please choose: rock (r) - paper (p) - scissors (s)\n");
+	char input;
+	int res;
+
+	do {
+		printf("Please choose: rock (r) - paper (p) - scissors (s)\n");
+		scanf("%1s", &input);
+		res = char_to_element_index(input);
+		if (!res)
+			printf("You've chosen something strange!\n");
```

Comment:
`char_to_element_index()` in case of error returns `UNKNOWN`, but return value is compared to `0`.

Currently `UNKNOWN == 0`, but this isn't guaranteed in case of modification. If return values have exact meaning, it's comparison should be explicit.